pkg/processor/metricfrequencyprocessor/pdata.go

```diff
@@ -0,0 +1,24 @@
+package metricfrequencyprocessor
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"sort"
+)
```

Comment:
```suggestion
import (
	"sort"

	"go.opentelemetry.io/collector/consumer/pdata"
)
```

Linter should complain about it 🤔  did you rebase ?

Reply:
no, perhaps linter is only active for PRs to master?

Reply:
good point, it is (+ release branches)

Reply:
Is there support in Goland for such convention?

Reply:
🤷‍♂️  but probably it's easy to enable. This repo is still not 100% but we'll be aiming for using https://pkg.go.dev/golang.org/x/tools/cmd/goimports?utm_source=godoc  + https://github.com/pavius/impi

Reply:
there is, fixed